templates/todo/projects/csharp-mongo/.devcontainer/Dockerfile

```diff
@@ -0,0 +1,7 @@
+ARG VARIANT=bullseye
```

Comment:
Seems like this is still the same for all templates.  We could keep it in common.

Reply:
Fixed in my commit